packages/dso-toolkit/components/Componenten/badge/badge.njk

```diff
@@ -1 +1,10 @@
 <span {{ className('dso-badge', [modifier, 'badge-' + modifier]) }}>{% if status -%}<span class="sr-only">{{ status }}: </span>{% endif -%}{{ label }}</span>
+
+<dso-badge
+  {{- attributes(
+    [status, ' status', status],
```

Comment:
Waarom staat die spatie daar? `' status'`

Reply:
Schijnbaar is die nodig, anders krijg je `<dsobadgestatus=...`, dat hoeft dus niet als er al een attribuut voor staat

Reply:
Nee die spatie is _niet_ nodig, er staat een `{{-` die de whitespace weg haalt. Deze is nu verwijderd.